opentelemetry-sdk/src/opentelemetry/sdk/_metrics/instrument.py

```diff
@@ -19,142 +19,142 @@
 # all instances. Implementations of these classes must not make any change to
 # this default dictionary in __init__.
 
+from opentelemetry._metrics.instrument import Counter as APICounter
+from opentelemetry._metrics.instrument import Histogram as APIHistogram
 from opentelemetry._metrics.instrument import (
-    Counter,
-    Histogram,
-    ObservableCounter,
-    ObservableGauge,
-    ObservableUpDownCounter,
-    UpDownCounter,
+    ObservableCounter as APIObservableCounter,
 )
-from opentelemetry.sdk._metrics.aggregation import (
-    ExplicitBucketHistogramAggregation,
-    LastValueAggregation,
-    SumAggregation,
+from opentelemetry._metrics.instrument import (
+    ObservableGauge as APIObservableGauge,
+)
+from opentelemetry._metrics.instrument import (
+    ObservableUpDownCounter as APIObservableUpDownCounter,
 )
+from opentelemetry._metrics.instrument import TCallback
+from opentelemetry._metrics.instrument import UpDownCounter as APIUpDownCounter
+from opentelemetry.sdk.util.instrumentation import InstrumentationInfo
 
 
 class _Instrument:
     def __init__(
         self,
-        name,
-        unit="",
-        description="",
-        aggregation=None,
-        aggregation_config={},
+        instrumentation_info: InstrumentationInfo,
```

Comment:
Nit: Can have we `InstrumentationInfo` as the last parameter?

Reply:
No, actually, it is a positional parameter, it has to be before the keyword arguments.